### ISSUE TITLE
[Accessibility] fig-alt does not work for mermaid

### ISSUE THREAD

**Issue by jooyoungseo** [quarto-dev/quarto-cli#3873]
Currently, `fig-alt` does not work for mermaid diagram. However, there seems a workaround. Mermaid has their own [accessibility options](https://mermaid.js.org/config/accessibility.html).

@cscheid -- We may want to bind `fig-cap` to `accTitle` and `fig-alt` to `accDescr`. Other additional accessibility fixes that I am not mentioning here need to be taken care of on mermaid side.

 ````

## Reprex

---
title: "Untitled"
format: html
---

# Mermaid Accessibility

## Inaccessible Version

```{mermaid}
%%| fig-alt: testing

flowchart LR
  A[Hard edge] --> B(Round edge)
  B --> C{Decision}
  C --> D[Result one]
  C --> E[Result two]
```


## Accessible Version

```{mermaid}
graph LR
      accTitle: Big Decisions
      accDescr: Bob's Burgers process for making big decisions
      A[Identify Big Descision] --> B{Make Big Decision}
      B --> D[Be done]
```

````

**Comment by cscheid** [comment]
Fantastic, thank you. You'll see in the code that I actually already have a todo for accessibility in `src/core/handlers/base.ts`, line 625. This should be fixed for all TS cell handlers, not only mermaid.

**Comment by cscheid** [comment]
Actually, I will fix this early on in our 1.4 version, since we'll have to upgrade to mermaid 10.0.0, and I don't want to go over the codebase twice.

**Comment by mcanouil** [comment]
Duplicate of (or the other way around):

- https://github.com/quarto-dev/quarto-cli/issues/9003

**Comment by mcanouil** [comment]
Currently, this means PDF UA-1 validation will fail for Typst when a diagram is used.
